structuretoolkit/analyse/spatial.py

```diff
@@ -2,9 +2,11 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
+from ase.atoms import Atoms
 import numpy as np
 from scipy.sparse import coo_matrix
 from scipy.spatial import ConvexHull, Delaunay, Voronoi
+from typing import List, Optional
```

Comment:
The ability to directly hint `list` instead of `typing.List` came with python 3.9. I see in the toml file that `structuretoolkit` actually claims to support back to 3.8. You already hint `list` (`dict`, ...) directly elsewhere, so converting everything to the <=3.8 style would be a lot of work. Further, in the unit tests I see that you don't actually test 3.8 anymore. So I'd suggest the fix-of-least-action is to always hint the new way here and drop 3.8 support.

```suggestion
from typing import Optional
```